gui_qt/project_state.py

```
@@ -391,6 +390,47 @@ def _write_json_object(self, path: Path, data: dict[str, Any]) -> None:
+
+    def clear_workspace_root(self) -> None:
+        """Clear configured workspace (registry features stay gated until set again)."""
+        self._workspace_root = None
+        if not self.config_path.exists():
+            return
+        try:
+            data = self._read_json_object(self.config_path, "translator_config.json")
+            data.pop("workspace_root", None)
+            self._write_json_object(self.config_path, data)
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)